xbmc/settings/MediaSettings.h

```diff
@@ -24,6 +24,8 @@
 
 class TiXmlNode;
 
+static constexpr float AUDIO_DELAY_STEP = 0.025f;
```

Comment:
maybe it would be nice if you could document this value so that we know what it refers to. But not a blocking change request.

Reply:
yes done